planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -2833,12 +2833,15 @@ double calcMinimumLaneChangeLength(
   const double & max_lateral_acc = lat_acc.second;
   const double & lateral_jerk = common_param.lane_changing_lateral_jerk;
   const double & finish_judge_buffer = common_param.lane_change_finish_judge_buffer;
+  const auto prepare_buffer = 0.5 * common_param.max_acc *
```

Comment:
For the length of the prepare segment, we use `prepare_length`. So could you change it to `prepare_length`?

Reply:
changed in [c4c1405](https://github.com/autowarefoundation/autoware.universe/pull/4242/commits/c4c140511a6a695224eef58b828a45c232dcfab4)